tst/schedulers/test_schedulers_api.py

```diff
@@ -65,6 +66,7 @@ def make_ray_skopt():
     SimpleScheduler(config_space=config_space, metric=metric1),
     RandomSearch(config_space=config_space, metric=metric1),
     BayesianOptimization(config_space=config_space, metric=metric1),
+    RegularizedEvolution(config_space=config_space, metric=metric1, mode='min'),
```

Comment:
Also why specifying min only in this case? For consistency with the surrounding code, I would keep the default mode.